bin/ch-pack-image

```diff
@@ -0,0 +1,79 @@
+#!/bin/sh
+
+libexec=$(cd "$(dirname "$0")" && pwd)/../libexec/charliecloud
+. "${libexec}/base.sh"
+
+# shellcheck disable=SC2034
+usage=$(cat <<EOF
+Flatten a builder image into a Charliecloud packed file.
+
+Usage:
+
+  $ $(basename "$0") [-b BUILDER] [--fmt CH_PACK_FMT] IMAGE OUTDIR [ARGS ...]
+
+You must have sufficient privilege (via sudo) to run the Docker commands.
+EOF
+)
+
+parse_basic_args "$@"
+
+# Kludgey argument parsing.
+while true; do
+    case $1 in
+        -b|--builder)
+            shift
+            CH_BUILDER=$1
+            shift
+            ;;
+        -b=*|--builder=*)
+            CH_BUILDER=${1#*=}
+            shift
+            ;;
+        --pack-fmt)
+            shift
+            CH_PACK_FMT=$1
```

Comment:
I think explicit variables are more clear, e.g. `CH_PACK_FORMAT`.

Reply:
I'm not opposed to this, but I also don't like how much work I would have to do to go change it everywhere lol